static/app/views/performance/browser/interactionSummary/useInteractionBreakdownTimeseriesQuery.tsx

```diff
@@ -0,0 +1,82 @@
+import {getInterval} from 'sentry/components/charts/utils';
+import {Series} from 'sentry/types/echarts';
+import EventView, {MetaType} from 'sentry/utils/discover/eventView';
+import {
+  DiscoverQueryProps,
+  useGenericDiscoverQuery,
+} from 'sentry/utils/discover/genericDiscoverQuery';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+
+type Props = {
+  element: string;
+  operation: string;
+  page: string;
+};
+
+export const useInteractionBreakdownTimeseriesQuery = ({
+  operation,
+  element,
+  page,
+}: Props): {data: Series[]; isLoading: boolean} => {
+  const pageFilters = usePageFilters();
+  const location = useLocation();
+  const organization = useOrganization();
+  const projectTimeSeriesEventView = EventView.fromNewQueryWithPageFilters(
+    {
+      yAxis: [`p75(transaction.duration)`],
+      name: 'Interaction Duration',
+      query: `transaction.op:${operation} transaction:${page} interactionElement:"${element.replaceAll(
+        '"',
+        '\\"'
+      )}"`,
```

Comment:
I think if you use `MutableSearch` to construct this filter it'll do the escaping for you!

Reply:
Nice, thanks for the suggestion! Changed it to use `MutableSearch` now